src/commands/manual-exec.ts

```diff
@@ -60,29 +70,44 @@ export async function manualExec(
     page: number
     wallet?: string
     solanaOfframp?: string
+    solanaRouter?: string
```

Comment:
Why `solanaRouter`? If this is on source, it's the OnRamp address, isn't it? I'd rather not add new cli parameters if we can auto-discover them